cmd/grafana-agent/Dockerfile

```diff
@@ -44,6 +44,15 @@ EOF
 COPY --from=build /src/agent/build/grafana-agent /bin/grafana-agent
 COPY cmd/grafana-agent/agent-local-config.yaml /etc/agent/agent.yaml
 
+ARG UID=0
+ARG GF_UID=473
+ARG GF_UNAME="grafana-agent"
```

Comment:
Nits:
- Should we put args higher up, under `FROM`?
- The names of args are not self-explanatory like the other args we use. Maybe add a few words of comment what these do?